### PR TITLE
Save respondent result

### PR DESCRIPTION
**Pull request by YuriiHoliuk**

### CODE REVIEW

File: lib/routes/quiz-result.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		if (!activation) {
+			ctx.status = 400;
+			ctx.body = 'No quiz with this ID';
+			return;
+		}
+
+		// because mongoose return Object with custom toString
+		const quiz = JSON.parse(JSON.stringify(activation.quiz));
+
+		const quizResult = QuizResult.parseResults(results, quiz);
```
Comment (PavelMelnyk) [inline lib/routes/quiz-result.js]: why not `this` ?

File: lib/routes/quiz-result.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+		}
+
+		if (!activation) {
+			ctx.status = 400;
+			ctx.body = 'No quiz with this ID';
+			return;
+		}
+
+		// because mongoose return Object with custom toString
+		const quiz = JSON.parse(JSON.stringify(activation.quiz));
```
Comment (pavlom0x) [inline lib/routes/quiz-result.js]: did you try `doc.toObject({ getters: true, virtuals: false })` ?

http://mongoosejs.com/docs/api.html#document_Document-toObject

**Review by tatsiak — DISMISSED** [review]
approve

File: lib/routes/quiz-result.js (hunk 2 above)
Comment (YuriiHoliuk) [reply lib/routes/quiz-result.js]: @PavelMelnyk thanx, this works pretty well

**Review by tatsiak — APPROVED** [review]
approve

**Pull request merged**